doc/articles/guides/status-bar-theme-color.md

```diff
@@ -0,0 +1,149 @@
+        </StackPanel>
+    </Grid>
+    ```
+3. Add a [`VisibleBoundsPadding.PaddingMask`](../features/VisibleBoundsPadding.md) to the root `<Grid>` to prevent the content being partially covered by the status and command bars:
+    ```xml
+    <Page ...
+          xmlns:toolkit="using:Uno.UI.Toolkit"
+          ...>
+
+        <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+              toolkit:VisibleBoundsPadding.PaddingMask="Top">
+    ```
+4. In `MainPage.xaml.cs`, expose the `MyCommandBar` which will be referenced in a later step:
+    ```cs
+    public static MainPage Instance { get; private set; }
+
+    public MainPage()
+    {
+        Instance = this;
+
+        this.InitializeComponent();
+    }
+
+    public CommandBar GetCommandBar() => MyCommandBar;
+    ```
+
+5. In `App.xaml.cs`, enable the native frame navigation to use the native style for `CommandBar`, instead of the UWP style:
+    ```cs
+    public App()
+    {
+        ConfigureFilters(global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory);
+
+    #if __IOS__ || __ANDROID__
+        // Enable native frame navigation.
+        Uno.UI.FeatureConfiguration.Style.ConfigureNativeFrameNavigation();
+    #endif
+
+        this.InitializeComponent();
+        this.Suspending += OnSuspending;
+    }
+    ```
+
+6. Subscribe to the `UISettings.ColorValuesChanged` event from `App.xaml.cs`:
+    > Note that the instance of `UISettings` is kept to prevent it from being disposed when going out of scope, which would otherwise dispose the event subscription.
+    ```cs
+    public sealed partial class App : Application
+    {
+        private UISettings _uiSettings;
+        private bool? _wasDarkMode = null;
+
+        /* ... */
+
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        {
+            /* existing code... */
+
+            // add this line at the end of OnLaunched:
+            ConfigureStatusBar();
+        }
+
+        /* ... */
+
+        private void ConfigureStatusBar()
+        {
+            // Listen for the system theme changes.
+            _uiSettings = new UISettings();
+            _uiSettings.ColorValuesChanged += (s, e) => UpdateStatusBar();
+
+    #if __IOS__
+            // Force an update when the app is launched.
+            UpdateStatusBar();
+    #endif
+        }
+    }
+    ```
+
+7. Implement the `UpdateStatusBar()` method:
+    ```cs
+    private void UpdateStatusBar()
+    {
+        // === 1. Determine the current theme from the background value,
+        // which is calculated from the theme and can only be black or white.
+        var backgroundColor = _uiSettings.GetColorValue(UIColorType.Background);
+        var isDarkMode = backgroundColor == Windows.UI.Colors.Black;
+
+    #if __ANDROID__
+        // Prevent deadlock as setting StatusBar.ForegroundColor will also trigger this event.
+        if (_wasDarkMode == isDarkMode) return;
+        _wasDarkMode = isDarkMode;
```

Comment:
Can it be done in `_uiSettings.ColorValuesChanged` instead ?

Reply:
like this a8caf72?